### PR TITLE
New strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: weblate_web/templates/base.html [hunk 1]
```diff
@@ -89,7 +89,7 @@
                 <a href="https://status.weblate.org/" class="service-status">{% trans "Service Status" %}</a>
             </div>
 			<div class="footer-left">
-                Copyright © 2012—2019 <a href="https://cihar.com/">Michal Čihař</a>  •  {% trans "The software is Open Source under GNU GPLv3" %}
+                Copyright © 2012—2019 <a href="https://cihar.com/">Michal Čihař</a>  •  {% trans "The software is copylefted libre software, under GNU GPLv3+ license" %}
```
Comment (nijel) [inline weblate_web/templates/base.html]: This string needs to be kept shorter, it's in page footer.

How about just mentioning the license in full detail?

```suggestion
                Copyright © 2012—2019 <a href="https://cihar.com/">Michal Čihař</a>  •  {% trans "The software is under GNU GPLv3+ license" %}
```

File: weblate_web/templates/download.html [hunk 2]
```diff
@@ -46,23 +46,23 @@ <h2 class="section-title left">{% trans "Cloud or virtualization images" %}</h2>
...
+                    <a href="https://docs.weblate.org/en/latest/admin/deployments.html#openshift" class="button">{% trans "OpenShift guide" %}</a>
                 </div>
                 <div class="col">
                     <div class="d-logo"><img src="{% static "img/weblate-stack.png" %}" alt="#" /></div>
-                    <div class="title">{% trans "BitNami Weblate stack" %}</div>
+                    <div class="title">{% trans "Bitnami Weblate stack" %}</div>
                     <p>{% trans "Native installation for most platforms and support for public cloud providers." %}</p>
                     <ul>
-                        <li><a href="https://bitnami.com/stack/weblate/cloud/google">Google Cloud Platform</a></li>
+                        <li><a href="https://bitnami.com/stack/weblate/cloud/google">Google Cloud platform</a></li>
```
Comment (nijel) [inline weblate_web/templates/download.html]: It's AFAIK called "Google Cloud Platform", see https://cloud.google.com/products/

File: weblate_web/templates/support.html [hunk 3]
```diff
@@ -137,13 +137,13 @@ <h2 class="section-title bigger">{% trans "Mailing lists" %}</h2>
...
                     <label>{% trans "E-mail" %}</label>
-                    <input type="email" name="email" class="mailing-text" placeholder="{% trans "Your e-mail..." %}" />
+                    <input type="email" name="email" class="mailing-text" placeholder="{% trans "Your e-mail…" %}" />
                     <input type="submit" class="mailing-submit" value="{% trans "Subscribe" %}" />
                 </form>
                 <ul>
                     <li>{% trans "<strong>Low traffic</strong> mailing list with announcements about premium Weblate services" %}</li>
-                    <li>{% trans "The list is moderated — only selected users can post announcements" %}</li>
-                    <li>{% blocktrans with url="https://lists.cihar.com/hyperkitty/list/hosted-weblate-announce@lists.cihar.com/" %}Browse <a href="{{ url }}">mailing list archives</a> to find answers{% endblocktrans %}</li>
+                    <li>{% trans "The list is moderated — only select users can post announcements" %}</li>
```
Comment (nijel) [inline weblate_web/templates/support.html]: Really? This doesn't sound right to me.

File: weblate_web/templates/download.html (hunk 2 above)
Comment (comradekingu) [reply weblate_web/templates/download.html]: I thought I had changed that back.

File: weblate_web/templates/support.html (hunk 3 above)
Comment (comradekingu) [reply weblate_web/templates/support.html]: It is a bit subtle, but "a select few" means the few that are selected, have some distinguishing feature, as opposed to "a selected few" could mean a narrow random assortment.
https://www.dailywritingtips.com/select-vs-selected/

File: weblate_web/templates/base.html (hunk 1 above)
Comment (comradekingu) [reply weblate_web/templates/base.html]: I went with "Software licensed GNU GPLv+3" Short and concise. It already scrolls past the other text
Skjermbilde fra 2019-07-16 19 53 26

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**